frontend/src/components/Cell.tsx

```diff
@@ -28,18 +28,23 @@ export const Cell = ({
         on {`${date}`}
       </label>
       <input
-        type="number"
+        type="text"
         id={`${topic.issue.id}${topic.activity.id}${formatDate(
           date,
           "yyyy-MM-dd"
         )}`}
-        min={0}
         onChange={(event: any) => {
+          //makes sure that users can only input positive numbers up to 999.99999999...
+          //with an unlimited number of decimals behind the delimiter
+          event.target.value = event.target.value
+            .replace(/[^0-9.]/g, "")
+            .replace(/(^\d{3})\d/g, "$1")
```

Comment:
```suggestion
            .replace(/(^\d{3})\d+/g, "$1")
```

Reply:
Without the + quantifier I can paste whichever number I want, no matter how long it is.

Reply:
Yes, as @jonandernovella says, but I would write it as

```
.replace(/^(\d{3})\d+/, "$1")
```

That is, move the `^` anchor outside of the capture group and drop the `g` flag (never needed with anchored expressions).

I'm also wondering whether the second `\d` should be `[^.]`, i.e. "any non-dot character".

Reply:
I added the + and removed the g.